networkx/algorithms/tree/mst.py

```diff
@@ -721,6 +722,228 @@ def maximum_spanning_tree(G, weight="weight", algorithm="kruskal", ignore_nan=Fa
     return T
 
 
+@py_random_state(3)
+def random_spanning_tree(G, weight=None, multiplicative=True, seed=None):
+    """
+    Sample a random spanning tree using the edges weights of the graph.
+
+    This function supports two different methods for determining the
+    probability of the graph. If `multiplicative` is `True`, the probability
+    is based on the product of edge weights, and if `multiplicative` is
+    `false` it is based on the sum of the edge weight. However, since it is
```

Comment:
Just a nit:

```suggestion
    This function supports two different methods for determining the
    probability of the graph. If ``multiplicative=True``, the probability
    is based on the product of edge weights, and if ``multiplicative=False``
    it is based on the sum of the edge weight. However, since it is
```